pkgs/by-name/vd/vdhcoapp/package.nix

```diff
@@ -63,6 +62,9 @@ buildNpmPackage rec {
 
   postInstall = ''
     $out/bin/vdhcoapp install
+    mkdir -p $out/lib/mozilla/native-messaging-hosts
+    cp -rf /build/.mozilla/native-messaging-hosts/net.downloadhelper.coapp.json $out/lib/mozilla/native-messaging-hosts/net.downloadhelper.coapp.json
+    echo "don't forget to add vdhcoapp to programs.firefox.nativeMessagingHosts.packages in your configuration.nix"
```

Comment:
this will be likely run on hydra for caching on the binary cache, not sure if it helps the user to dig thru the hydra logs to find that.